index.js

```diff
@@ -3,14 +3,24 @@
  */
 
 var RNDeviceBrightness = require('react-native').NativeModules.RNDeviceBrightness;
+var Platform = require('react-native').Platform;
 
 module.exports = {
   setBrightnessLevel: function(brightnessLevel) {
     if (brightnessLevel < 0 || brightnessLevel > 1) {
-      throw Error('⚠️ BrightnessLevel value must betweens 0 to 1 ⚠️');
+      if(!(Platform.OS === 'android' && brightnessLevel===-1)) {
+          throw Error('⚠️ BrightnessLevel value must betweens 0 to 1 ⚠️');
+      }
     }
 
     RNDeviceBrightness.setBrightnessLevel(brightnessLevel);
   },
   getBrightnessLevel: RNDeviceBrightness.getBrightnessLevel,
+
+  getSystemBrightnessLevel: function () {
+    if(Platform.OS!=='android'){
+        throw Error('⚠️ android only supported ⚠️');
```

Comment:
According the [Airbnb Javascript Style Guide -
Whitespace & Equality](https://github.com/airbnb/javascript#whitespace)
Should reduce two spaces here.